docs/reference/run.md

```diff
@@ -629,6 +631,12 @@ command:
 
     $ docker run --security-opt label:type:svirt_apache_t -it centos bash
 
+If you want to prevent your container processes from gaining additional
+privileges, you can execute the following command:
```

Comment:
Does this need more info on what is meant with "gaining additional privileges", or is there a really simple example we can use to how what it's preventing, below?